demo.adoc

```diff
@@ -39,7 +39,9 @@ auth0 test token -a https://<your-auth0-domain>/api/v2/ -s openid
 
   TOKEN=eyJraWQiOiJYa2pXdjMzTDRBYU1ZSzNGM...
 
-== Make a Java REST API with Micronaut
+== Make a Reactive Java REST API with Micronaut
+
+Nothing needs to be done to make Micronaut reactive. It has no separate reactive web framework. See Piotr Minkowski's https://piotrminkowski.com/2019/11/12/micronaut-tutorial-reactive/[Micronaut Tutorial: Reactive] and https://guides.micronaut.io/latest/tag-reactive.html[Micronaut's Reactive Guides] for more information.
```

Comment:
@ sdelamo Can you please confirm this is truthful? 

Related: can you confirm that Micronaut enables virtual threads by default?

Reply:
If you use the Netty runtimes and you have a controller method which blocks (e.g. it calls a database with a blocking driver) then you should annotate your controller method with `@ExecuteOn(BLOCKING)`. If you use a Java version with supports virtual threads it uses virtual threads.